src/node-types.json

```diff
@@ -1399,11 +1403,11 @@
   },
   {
     "type": "file",
-    "named": true
+    "named": false
```

Comment:
Is this (and the change for pattern below) related to the addition of `&ordered` 🤔